exporters/sender/okhttp/src/test/java/io/opentelemetry/exporter/sender/okhttp/internal/OkHttpSenderTest.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.exporter.sender.okhttp.internal;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import io.opentelemetry.exporter.internal.InstrumentationUtil;
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.atomic.AtomicBoolean;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+abstract class OkHttpSenderTest<T> {
+
+  @BeforeEach
+  void setUp() {
+    OkHttpUtil.propagateContextInDispatcher = true;
+  }
+
+  @AfterEach
+  void tearDown() {
+    OkHttpUtil.propagateContextInDispatcher = false;
+  }
+
+  @Test
+  void testSuppressInstrumentation() throws InterruptedException {
+    CountDownLatch lock = new CountDownLatch(1);
```

Comment:
Nit: call the latch a latch not a lock. 😁

Reply:
Thanks, it's updated now